rrrspec-web/Gemfile

```diff
@@ -3,4 +3,5 @@ source 'https://rubygems.org'
 gem 'rrrspec-client', path: '../'
 gem 'rrrspec-server', path: '../'
 gemspec
+gem 'redis', '~> 3.3.5'
```

Comment:
I bet this should be in rrrspec-client.gemspec and rrrspec-server.gemspec. Why do you add the constraint only in rrrspec-web/Gemfile?

Reply:
You are right. redis gem are in gemspecs of both rrrspec-client and rrrspec-server. Because I was not sure if they should be updated. Should I update them?

Reply:
What do you mean by that? Do you want to know why I specified version of redis gem? If so, it is because of antirez/redis#4272.

Reply:
1. These constraints must be declared in gemspec (because `rrrspec-*/` are implemented as a rubygem)
2. If we're affected the issue you've present in the issue description and your last comment, we need to declare the constraint (`~> 3.3.5`)
   - `redis` dependency comes from rrrspec-client.gemspec
   - rrrspec-web itself doesn't directly depend on Redis gem; See `git grep 'redis\|Redis' rrrspec-web/`
   - I bet the code which affects the issue exists in rrrspec-client
   - Thus the constraint need to be declared in rrrspec-client.gemspec

Reply:
That's why I've asked **Where** is a code affected by the issue.

Reply:
Thank you for the explanation and the additional fix :sparkles: